src/Orc.NuGetExplorer/Configuration/NuGetSettings.cs

```diff
@@ -309,6 +319,8 @@ private void SetNuGetValues(string section, string subsection, IList<AddItem> va
 
         private void UpdateKeyList(IList<AddItem> values, string valuesListKey)
         {
+            ArgumentNullException.ThrowIfNull(() => values);
```

Comment:
Please search for all "ArgumentNullException.ThrowIfNull(() =>" and make sure to correctly replace.